pkg/apiserver/registry/inventory/virtualmachine/rest_test.go

```diff
@@ -113,7 +113,6 @@ var _ = Describe("Virtual Machine", func() {
 	}
 
 	Describe("Test Get function of Rest", func() {
-		By("Test Get function of Rest")
```

Comment:
By is supported. Please comment why By field is removed from all the unit test and integration tests.

Reply:
In the unit test By is used mostly in Describe but that is not allowed and also the description and By statement is same. Its kind of repetitive.

So I thought we can follow the same in integration test as well. By statement prints only when the issue occurs. Thats why added logs for that.

Reply:
By is also not supported in BeforeSuite(), Context(), while it can be used in It(), BeforeEach(), AfterEach(). So please add the By field back wherever it is applicable. It makes easier to visualize the logs and debug.